src/lib.rs

```diff
@@ -1138,6 +1138,14 @@ impl GooseAttack {
                 });
             }
 
+            if self.configuration.no_reset_stats {
+                return Err(GooseError::InvalidOption {
+                    option: "--no-reset-stats".to_string(),
+                    value: self.configuration.no_reset_stats.to_string(),
+                    detail: Some("--no-reset-stats is only available to the manager".to_string()),
```

Comment:
Q: Is it better to say `to the manager` or `for the manager`?

Reply:
IMO `to the manager`, or possibly `on the manager`.  The existing verbiage shows up 10 other times in the code base, for other flags and options.

Reply:
Maybe: Can only be set on the manager

Reply:
https://github.com/tag1consulting/goose/issues/132